examples/seismic/acoustic/operators.py

```diff
@@ -161,13 +195,19 @@ def GradientOperator(model, geometry, space_order=4, save=True,
 def BornOperator(model, geometry, space_order=4,
                  kernel='OT2', **kwargs):
     """
-    Constructor method for the Linearized Born operator in an acoustic media
+    Construct an Linearized Born operator in an acoustic media.
+
+    Parameters
+    ----------
+    model : Model
+        Object containing the physical parameters.
+    geometry :
+
+    space_order : int, optional
+        Space discretization order.
+    kernel :
+        Type of discretization, centered or shifted.
```

Comment:
`OT2` or `OT4` shifted, centered, .. .is for TTI

Reply:
Is is fine like : 
   kernel : OT2 or OT4
        Type of discretization, centered or shifted.

Reply:
see my comment above

Reply:
fixed